distro/config.go

```diff
@@ -75,40 +77,26 @@ type config struct {
 	Propagator propagation.TextMapPropagator
 	SpanLimits *trace.SpanLimits
 
-	ExportConfig       *exporterConfig
-	TracesExporterFunc traceExporterFunc
+	ExportConfig        *exporterConfig
+	TracesExporterFunc  traceExporterFunc
+	MetricsExporterFunc metricsExporterFunc
 }
 
 // newConfig returns a validated config with Splunk defaults.
 func newConfig(opts ...Option) *config {
 	c := &config{
 		Logger:     logger(zapConfig(envOr(otelLogLevelKey, defaultLogLevel))),
+		Propagator: autoprop.NewTextMapPropagator(),
```

Comment:
The option for propagator has been removed before 1.0.0 to reduce the API surface.

I was thinking about doing this for logger, but then I realized that it may be safer to initialize the logger beforehand as some options may log things when there is an error.